easybuild/easyconfigs/i/intel-compilers/intel-compilers-2021.1.eb

```diff
@@ -0,0 +1,21 @@
+name = 'intel-compilers'
+version = '2021.1'
+
+homepage = 'https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit.html'
+description = "Intel C, C++ & Fortran compilers (classic and oneAPI)"
+
+toolchain = SYSTEM
+
+# see https://software.intel.com/content/www/us/en/develop/tools/oneapi/hpc-toolkit/download.html
+source_urls = ['https://registrationcenter-download.intel.com/akdlm/irc_nas/17427/']
+sources = ['l_HPCKit_p_%(version)s.0.2684_offline.sh']
```

Comment:
I would just hardcode the version for hpckit because it is not the same as the compiler.

Reply:
Currently, you have to set the LD_LIBRARY_PATH to invoke the compiler so it can find libimf. I think this is a bug and the need to set LD_LIBRARY_PATH will go away. I believe it will be harmless to continue setting the LD_LIBRARY_PATH. You might want to keep this in mind for future changes.

Reply:
That's handled by the easyblock in https://github.com/easybuilders/easybuild-easyblocks/pull/2305, but thanks for bringing that up!